src/View/Helper/FormMultiCheckbox.php

```diff
@@ -412,6 +413,7 @@ protected function getInputHelper(): FormInput
             return $this->inputHelper;
         }
 
+        assert($this->view !== null);
```

Comment:
I think baselining these would be more appropriate, as @gsteel keeps working on cleaning up the abstract helpers 🤔